scrapyd_k8s.sample-docker.conf

```diff
@@ -5,9 +5,9 @@
 bind_address = 0.0.0.0
 http_port    = 6800
 
-## uncomment this section to enable authorization
-# username = questionmark
-# password = changeme123
+## Uncomment this section to enable authorization
+#username    = sampleusername
+#password    = changeme123
 
```

Comment:
Please really look at the surrounding style and stick to it (and if you suggest to change the style, discuss it).
In this cases, that would be single `##` also for the comment.
Other comments also end with a dot `.`, so that would be good to use here as well.

Could you use as comment: `# Optional basic http authentication for all endpoints.` ?